db/migrate/20160104131622_remove_deprecated_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveDeprecatedColumns < ActiveRecord::Migration
+  def change
+    remove_column :subject_workflow_counts, :set_member_subject_id
+    remove_column :classifications, :subject_ids
```

Comment:
one last question...removing this column will still allow the classification -> subjects association to build correctly right?